requirements.txt

```diff
@@ -0,0 +1,7 @@
+BeautifulSoup==3.2.1
+dj-database-url==0.4.1
```

Comment:
dj-database-url is a utility that has been used for database purposes, it basically wil help in loading database into my dictionary from the DATABASE_URL environment variable.